rfc002-procope-blog.md

```diff
@@ -0,0 +1,208 @@
+
+They're periodical content, to keep engagement up. They're individually credited, and live on a GDP-branded Medium.com publication.
+
+To contrast, **white papers** would cover topics core to Good Docs Project templates, in philosophy and execution. They would be created by community consensus and represent the community as a unified element. They would live on the core GDP website, rather than in the publication models outlined below.
+
+White papers are created as needs arise, rather than an ongoing periodical model to keep external engagement up.
+
+The Procope team will be very happy to be involved with white papers, including co-running it to lend editorial expertise, but doesn't want to muddy this proposal by including a different sort of publication.
+
+#### Out of scope for now: templates for this content
+
+Templates are great! That's why GDP exists. But rather than starting with blog post templates, we want to give freedom for various forms of conversational writing.
+
+That said, some months in we do want to examine our content for templating, as that empowers onboarding contributing.
+
+### Content types
+
+All media is community-contributed pieces on documentation philosophy, best practices, and related opinion or research pieces. They are individually created and driven, representing a viewpoint within the community that the community respects enough to be associated with it, but with bylines that attach the viewpoint to the original author or authors.
+
+(This means the authors and the community co-own any potential fallout from poorly received posts or videos.)
+
+Blog posts are 500- to 1000-word pieces about a subject, whether that's a single topic, a top N post, a collection of interesting links with commentary, and so on. Some posts will be solidary, and others will be part of a series.
+
+Videos are 3- to 10-minute pieces about a subject, illustrating a narrative that helps those more engaged in visual narratives buy into our concepts and templates.
+
+#### Where content would live
+
+Blog posts would live on a Medium.com magazine, branded as part of the Good Docs Project. Medium.com is a known place where people can subscribe to stories, comment, and otherwise have a sense of expectation regarding a look and feel. I've taken the liberty of (creating a blank publication)[https://medium.com/the-good-docs-project], to make sure we have the name.
+
+Videos would live on YouTube, with icon and naming showing it as part of the Good Docs Project. YouTube is the obvious place, because hosting video content is expensive and YouTube has tools to help with ancillary needs. I've taken the liberty of (creating a blank channel)[https://www.youtube.com/channel/UCi01Sf06CeUeW5XlwqyNIpQ].
+
+*Aside:* We understand there's a blog component to the GDP website. We think that may be a good place to explore hosting white papers, to keep white papers directly attached to the material they support.
```

Comment:
I propose that this is also where the source of the blog content is stored as well.  It wouldn't hurt anything to have it sourced on the site and republished on any other platform that might extend the reach of the material.  There is also no mention in this section about medium.com requiring membership for access to content (after some small number of free reads).  I think that GDP content and materials should be free to access for everyone as a minimum.

Reply:
There's a hard no on republishing. That's additional work at the community management level.

Reply:
And depending how such an integration would be proposed, additional work in reformatting articles for another platform. As well as in propogating updates, should we have to make any copy or substantive editorial changes.

Reply:
(Also, I apologise if I sound too aggressive with "hard no." I'm trying to be vigilant about not overly adding time publishers need to handle the mechanics of publishing/verifying/editing content, as it's an already time-intensive role. Same with community management.) :)

Reply:
A thought I have after looking into the Medium.com paywall, after a few months of getting some analytics in on blog usage, we might play with a delayed manual syndication to a Medium publication, and see if that reaches significantly more people.

Said delay gives the benefit of subscribing to the core blog, and it allows for any updates happening within the first week of a post going life for whatever reason to be part of that syndicated post. (Having to edit in multiple places in response to a media crisis—like some Twitter outcry—is why I want to avoid real-time syndication.)

*If* we did that route, our social media would socialize our blog at time of publication, and we'll have some places where we'd let users know they can do the Medium thing if that's their preferernce, with the Medium posts always referring back to the original publication.